packages/packages_test.go

```diff
@@ -350,6 +384,60 @@ func TestPackagesFilter(t *testing.T) {
 				{Name: "etcd", Version: "1.0.0-rc2"},
 			},
 		},
+		{
+			Title: "non existing capabilities search",
+			Filter: Filter{
+				Capabilities: []string{"no_match"},
+			},
+			Expected: []filterTestPackage{
+				{Name: "apache", Version: "1.0.0"},
+				{Name: "nginx", Version: "2.0.0"},
+				{Name: "redisenterprise", Version: "1.0.0"},
+			},
+		},
+		{
+			Title: "observability capabilities search",
+			Filter: Filter{
+				Capabilities: []string{"observability"},
+			},
+			Expected: []filterTestPackage{
+				{Name: "apache", Version: "1.0.0"},
+				{Name: "nginx", Version: "2.0.0"},
+				{Name: "redisenterprise", Version: "1.0.0"},
+				{Name: "obs_package", Version: "1.1.0"},
+				{Name: "obs_sec_package", Version: "2.0.0"},
+				{Name: "obs_sec_uptime_package", Version: "2.0.0"},
```

Comment:
These packages shouldn't appear here, because they require both observability and security, and only observability is provided here.
```suggestion
				{Name: "obs_sec_package", Version: "2.0.0"},
				{Name: "obs_sec_uptime_package", Version: "2.0.0"},
```